server/api.go

```diff
@@ -160,9 +164,43 @@ func (p *MatterpollPlugin) handleEndPoll(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	p.postEndPollAnnouncement(request, poll.Question)
+
 	writePostActionIntegrationResponse(w, response)
 }
 
+func (p *MatterpollPlugin) postEndPollAnnouncement(request *model.PostActionIntegrationRequest, question string) {
+	team, err := p.API.GetTeam(request.TeamId)
+	if err != nil {
+		p.API.LogError(endPollAnnouncementPostError, "details", fmt.Sprintf("failed to GetTeam with TeamId: %s", request.TeamId))
+		return
+	}
+	permalink := fmt.Sprintf("%s/%s/pl/%s", *p.ServerConfig.ServiceSettings.SiteURL, team.Name, request.PostId)
+
+	pollPost, err := p.API.GetPost(request.PostId)
+	if err != nil {
+		p.API.LogError(endPollAnnouncementPostError, "details", fmt.Sprintf("failed to GetPost with PostId: %s", request.PostId))
+		return
+	}
+	channelID := pollPost.ChannelId
+
+	endPost := &model.Post{
+		UserId:    request.UserId,
+		ChannelId: channelID,
+		RootId:    request.PostId,
+		Message:   fmt.Sprintf(endPollSuccessfullyFormat, question, permalink),
+		Type:      model.POST_DEFAULT,
+		Props: model.StringInterface{
+			"override_username": responseUsername,
+			"override_icon_url": fmt.Sprintf(responseIconURL, *p.ServerConfig.ServiceSettings.SiteURL, PluginId),
+			"from_webhook":      "true",
+		},
+	}
+	if _, err := p.API.CreatePost(endPost); err != nil {
```

Comment:
Nit pick: Can we stay to our usual pattern here and exclude `_, err := p.API.CreatePost(endPost)` for if

Reply:
👍 
Are you saying this fix, right?
https://github.com/matterpoll/matterpoll/pull/46/commits/805f4a62322d9398264b77d83011abbd853d756c